libzhl/functions/Music.zhl

```diff
@@ -46,6 +46,16 @@ struct Music {
 		*(int*)((char*)this + 0x348) = 0;
 		*(int*)((char*)this + 0x364) = -99;
 	};
+	
+	inline bool ValidateMusicID(int id, int& max) {
```

Comment:
You should declare a dummy class in ZHL (MusicInstance ?) with a fixed size of 96, and add two pointers to said class in the Music class so you can use types and offsets instead of translating the ASM into C

Reply:
Added _minBound and _maxBound to Music, moved function over to IsaacRepentance_static.cpp

Reply:
Realized now this isn't quite what you meant. Going to try documenting this class a bit

Reply:
Now has a proper class, with a vector for it added to Music and used for this function